sky/core.py

```diff
@@ -589,16 +590,19 @@ def cancel(
         sky_logging.print(f'{colorama.Fore.YELLOW}'
                           f'Cancelling all jobs on cluster {cluster_name!r}...'
                           f'{colorama.Style.RESET_ALL}')
-        job_ids = None
+    elif not job_ids:
```

Comment:
For job_ids, do we mean is not None or is not empty list?

Reply:
Yep, added a comment for it. : )